scripts/utils.sh

```diff
@@ -17,6 +17,7 @@ set_env(){
     # performance tuning
     export VLLM_DELAYED_SAMPLING=${VLLM_DELAYED_SAMPLING:-"true"}
     export VLLM_ZERO_PADDING=${VLLM_ZERO_PADDING:-"true"}
+    export VLLM_CONFIG_HIDDEN_LAYERS=${VLLM_CONFIG_HIDDEN_LAYERS:-"1"}
```

Comment:
Is VLLM_CONFIG_HIDDEN_LAYERS name defined by us or the global habana branch? IMO, this naming is very confusing... maybe a better one should be VLLM_CONFIG_HIDDEN_LAYERS_MARK_STEP_INTERVAL.

Reply:
Yes, this is quite confusion. I removed it from `utils.sh` and plan to refactor the code and doc for `VLLM_CONFIG_HIDDEN_LAYERS`.